hydro_cli_examples/examples/topolotree/random_insertions.py

```diff
@@ -0,0 +1,61 @@
+import random
+import numpy as np
+
```

Comment:
You should include a comment at the top describing what the script does and how we are using it. It will make it easy for people to see if the scripts can be reused or improved on later on in the project.